hash/neocd.xml

```diff
@@ -329,6 +328,27 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<!--
+	source: redump.org/disc/80699/
+	<rom name="Bang^2 Busters (France) (En,Ja) (Unl).cue" size="339" crc="6e420f96" sha1="5e26f9c70009278f74bbd9f5db9cb5cdc4f2ef77"/>
+	<rom name="Bang^2 Busters (France) (En,Ja) (Unl) (Track 1).bin" size="7883904" crc="f555da2f" sha1="d9e54974652b9fbbe6c3ac82a70e9c06cc16700f"/>
+	<rom name="Bang^2 Busters (France) (En,Ja) (Unl) (Track 2).bin" size="9988944" crc="77311095" sha1="9b77e1ec7e308a567828b5bf8c1e57147781b9ab"/>
+	<rom name="Bang^2 Busters (France) (En,Ja) (Unl) (Track 3).bin" size="9988944" crc="898b1a26" sha1="771dfb863f36ab898bc7251abe67529d2060a282"/>
+	-->
+	<software name="bbbuster">
+		<description>Bang Bang Busters (France)</description>
```

Comment:
Why is this treated as a French release?

Reply:
Same criteria used for the "Treasure of the Caribbean (France)" published by the same company.
What do you suggest? Replace "France" by "Europe" on both games?

Reply:
Well, the fact that it supports English and Japanese language suggests it was intended to be a worldwide release.

Did the company actually attempt to restrict distribution?

Reply:
I didn't find any information about region restriction release.
So, I removed the "France" indication.